### ISSUE TITLE
EMLParty model unnecessarily changes XML IDs

### ISSUE THREAD

**Issue by amoeba** [NCEAS/metacatui#1430]
When editing an EML doc that has `id` attributes on a `<creator>` element, e.g.,

```xml
  <creator id="example.person">
    <individualName>
      <givenName>Example</givenName>
      <surName>Person</surName>
```

I ended up getting errors on save about missing references. It turns out this document had internal references such as:

```xml
<annotation references="example.person">
```

and the editor's EMLParty model was always overwriting whatever was already being used as the `creator` element's `id`. You can see from that code why this is called every time we run `updateDOM` (which is about five times per serialize) but, interestingly, you can see that we had code that did something entirely different in here before:

https://github.com/NCEAS/metacatui/blob/92edadcafcbb04dec6ddc8e24b48cc80fff79051/src/js/models/metadata/eml211/EMLParty.js#L596-L600

It seems to me we should at least re-use an `id` we get on parse so this call to `createID` should be guarded by a check for an existing id and not be run if one is found. Does anyone know the history of this spot in the code and see any reason we shouldn't re-use existing IDs instead of creating new ones?

**Comment by laurenwalker** [comment]
I agree that we should be reusing IDs if they are already there. I don't remember any discussions about why we chose to do it this way - it may have been simply overlooked. Thanks for the catch.
